core/jobs_registry.py

```diff
@@ -91,7 +92,9 @@
     user_jobs_one_off.UsernameLengthAuditOneOffJob,
     user_jobs_one_off.UsernameLengthDistributionOneOffJob,
     exp_jobs_one_off.MathExpressionInputInteractionOneOffJob,
-    exp_jobs_one_off.MultipleChoiceInteractionOneOffJob
+    exp_jobs_one_off.MultipleChoiceInteractionOneOffJob,
+    suggestion_jobs_one_off.SuggestionMathMigrationOneOffJob,
+    suggestion_jobs_one_off.SuggestionMathRteAuditOneOffJob
```

Comment:
Add this in sorted order (helps in finding the job)

Reply:
done